client/src/Account/components/Account.tsx

```diff
@@ -13,14 +13,17 @@ import { Spinner } from 'common/components'
 // layout
 import { NotFound } from 'layout/components'
 import useMediaQuery from 'common/hooks/useMediaQuery'
+import { accountConverter } from 'common/helpers/accountConverter'
 
 const Account: FC = () => {
   const { accountId } = useParams()
 
+  const address = accountConverter(accountId || '')
```

Comment:
very minor: maybe `convertedAddress` is a better variable name? since there is already an address (`id` property) in the `account`